src/scripts/changeTracker.ts

```diff
@@ -1,26 +1,25 @@
-// @ts-nocheck
-
+import type { ComfyApp } from "./app";
 import { api } from "./api";
 import { clone } from "./utils";
+import { ComfyWorkflow } from "./workflows";
```

Comment:
There is also a `ComfyWorkflow` defined under `types/comfyWorkflow`. I think it would be nice that we agree on a solution to distinguish these 2 symbols.

I would propose change `types/comfyWorkflow::ComfyWorkflow` to `ComfyWorkflowJSON`. WDYT?